TACC/Core-Portal
# FP-251: add Ticket ID to Ticket Modal Header

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [FP-251](https://jira.tacc.utexas.edu/browse/FP-251)

## Summary of Changes: ##
Added Ticket ID to the header of the Ticket modal and updated text color

## Testing Steps: ##
1. Run the branch and click on a ticket to view the modal.  The ticket ID should be in the top header in a light gray color to the left of the ticket subject in black.

## Notes: ##
